Objective-C/CBLDefaults.h

```diff
@@ -0,0 +1,73 @@
+//
+//  CBLDefaults.h
+//  CouchbaseLite
+//
+//  Copyright (c) 2022-present Couchbase, Inc All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+// THIS IS AN AUTOGENERATED FILE, MANUAL CHANGES SHOULD BE EXPECTED TO
+// BE OVERWRITTEN
+
+#import "CBLReplicatorTypes.h"
+
+/** [NO] Plaintext is not used, and instead binary encoding is used in log files */
+extern const BOOL kCBLDefaultLogFileUsePlainText;
```

Comment:
I think it's better to group them into sections using configuration class names so that it's easier to know which ones are for which configuration such as 

#pragma mark - CBLReplicatorConfiguration